.drone.star

```diff
@@ -410,7 +410,7 @@ def generate(compiler_ranges, cxx_range, max_cxx=2, coverage=True, docs=True, as
 
         if cache_dir != None and image_supports_caching(image, compiler):
             environment['drone_cache_mount'] = cache_dir
-            if image in images_used:
+            if image in images_used or buildtype != "boost":
```

Comment:
Who's expected to build the original cache on which we "git pull" or "submodule update" for the first time?

Don't we still need `drone_cache_rebuild` in one job to upload the new libs with "git pull" or "submodule update" work?

We "git pull" or "submodule update" do not do anything, can we set `drone_cache_rebuild` from `true` to `false` to skip the upload step?

Reply:
@alandefreitas
notice in this modification, 

> if image in images_used or buildtype != "boost":

"!=" instead of a "=". The effect is excluding "docs" from being the rebuilder instance. "boost" will rebuild.

> Who's expected to build the original cache on which we "git pull" or "submodule update" for the first time?

common_install.sh runs every time, regardless, and creates the original boost-root folder.

> Don't we still need drone_cache_rebuild 

drone_cache_rebuild is still present. Hasn't been removed from the procedure.

> When "git pull" or "submodule update" do not do anything, can we set drone_cache_rebuild from true to false to skip the upload step?

this is the same issue that has been present for a while. It only affects the rebuilder instances, and only takes 5 seconds. Not so bad. I will open an issue with meltwater but probably we can live with that delay.

Reply:
>  issue with meltwater

actually, I will delay asking them, because it's likely beyond their ability to fix right now, I have already opened an issue to be able to cache absolute paths instead of relative paths which would allow boost-root to be immediately cached instead of copying it back and forth.

Reply:
Sounds good